src/megatron/bridge/recipes/qwen_vl/qwen25_vl_dataset.py

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 from dataclasses import dataclass
-from typing import Any, Dict, Optional, Tuple
+from typing import Any, Dict, Tuple
```

Comment:
we should not rely on `Dict` or `Tuple` in this file anymore - they should be replaced by `dict` and `tuple`